src/structures/Utils.ts

```diff
@@ -163,41 +176,67 @@ export abstract class TrackUtils {
   static async getClosestTrack(
     unresolvedTrack: UnresolvedTrack
   ): Promise<Track> {
-    if (!TrackUtils.manager) throw new RangeError("Manager has not been initiated.");
+    if (!TrackUtils.manager)
+      throw new RangeError("Manager has not been initiated.");
 
     if (!TrackUtils.isUnresolvedTrack(unresolvedTrack))
       throw new RangeError("Provided track is not a UnresolvedTrack.");
 
-    const query = [unresolvedTrack.author, unresolvedTrack.title].filter(str => !!str).join(" - ");
-    const res = await TrackUtils.manager.search(query, unresolvedTrack.requester);
-
-    if (res.loadType !== "SEARCH_RESULT") throw res.exception ?? {
-      message: "No tracks found.",
-      severity: "COMMON",
-    };
+    const query = [unresolvedTrack.author, unresolvedTrack.title]
+      .filter((str) => !!str)
+      .join(" - ");
+    const res = await TrackUtils.manager.search(
+      query,
+      unresolvedTrack.requester
+    );
+
+    if (res.loadType !== "SEARCH_RESULT")
+      throw (
+        res.exception ?? {
+          message: "No tracks found.",
+          severity: "COMMON",
+        }
+      );
 
     if (unresolvedTrack.author) {
-      const channelNames = [unresolvedTrack.author, `${unresolvedTrack.author} - Topic`];
+      const channelNames = [
+        unresolvedTrack.author,
+        `${unresolvedTrack.author} - Topic`,
+      ];
 
-      const originalAudio = res.tracks.find(track => {
+      const originalAudio = res.tracks.find((track) => {
         return (
-          channelNames.some(name => new RegExp(`^${escapeRegExp(name)}$`, "i").test(track.author)) ||
-          new RegExp(`^${escapeRegExp(unresolvedTrack.title)}$`, "i").test(track.title)
+          channelNames.some((name) =>
+            new RegExp(`^${escapeRegExp(name)}$`, "i").test(track.author)
+          ) ||
+          new RegExp(`^${escapeRegExp(unresolvedTrack.title)}$`, "i").test(
+            track.title
+          )
         );
       });
 
-      if (originalAudio) return originalAudio;
+      if (originalAudio) {
+        if (unresolvedTrack.spotifyUri)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585532)
```suggestion
        if (unresolvedTrack.spotifyUri)
```